pkgs/development/interpreters/python/hooks/setuptools-check-hook.sh

```diff
@@ -6,7 +6,7 @@ setuptoolsCheckPhase() {
     runHook preCheck
 
     cp -f @setuppy@ nix_run_setup
-    @pythonCheckInterpreter@ nix_run_setup test
+    @pythonCheckInterpreter@ nix_run_setup test "${setuptoolsCheckFlagsArray[@]}"
```

Comment:
@FRidh do we need documentation for this one?

Reply:
There is a `### Setup hooks` in `python.section.md`. Probably good to mention it there.